src/style.css

```diff
@@ -1 +1,86 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  /* margin-inline: 50px; */
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+@media (min-width: 1024px) {
+  .header {
+    margin-inline: auto;
+    max-width: 924px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .header {
+    margin-inline: auto;
+    max-width: 1100px;
+  }
+}
+
+.logo__link {
+  text-decoration: none;
+  color: #000;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav__item {
+  margin-inline-end: 20px;
+}
+
+.nav__item:last-of-type {
+  margin-inline-end: 0;
+}
+
+.nav__link {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
```

Comment:
add `font-family` style to `body` or `html` selector. then you can use `font-family: inherit`
do not forget to add more fallback fonts
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/9657bb45-81df-4493-8651-7ab4f63c692f)